src/audio/buffer.c

```diff
@@ -228,11 +228,13 @@ void comp_update_buffer_produce(struct comp_buffer __sparse_cache *buffer, uint3
 
 	/* return if no bytes */
 	if (!bytes) {
+#if CONFIG_SOF_LOG_DBG_BUFFER
```

Comment:
it build nothing as default with buf_dbg, if change build log level, then it will print lot of log messages,  I just mean default there is no cycle cost, and enable it indeed cost too much, not sure need a config to just cover is expensive or not.